lib/ret_web/controllers/api/v2/credentials_controller.ex

```diff
@@ -0,0 +1,132 @@
+defmodule RetWeb.Api.V2.CredentialsController do
+  use RetWeb, :controller
+
+  alias Ret.{Repo}
+  alias Ret.Api.Credentials
+  alias Ecto.Changeset
+
+  import Ret.Api.TokenUtils,
+    only: [to_claims: 2, authed_create_credentials: 2, authed_list_credentials: 2, authed_revoke_credentials: 2]
+
+  # Limit to 1 TPS
+  # TODO : I copied this from other controllers. Is this an appropriate limit? -John
+  plug(RetWeb.Plugs.RateLimit when action in [:create, :update])
+
+  def index(conn, %{"app" => _anything} = _params) do
+    handle_list_credentials_result(conn, authed_list_credentials(Guardian.Plug.current_resource(conn), :app))
+  end
+
+  def index(conn, _params) do
```

Comment:
Should I add a parameter to specify an `account_id` in case an admin account wants to query for the tokens of a non-admin account?

Reply:
I can do this in a followup PR if we need it.